packages/web/src/scss/settings/_utilities.scss

```
@@ -1,25 +1,49 @@
 @use 'sass:map';
 @use '@tokens' as tokens;
+@use '../tools/utilities' as tools-utilities;
```

```suggestion
@use '../tools/utilities';
```
Should work too, right? 🤔 But I am ok with named import too.

---

I wanted to distinguish utilities from tools directory between this file.